test/orchard/java_test.clj

```diff
@@ -66,6 +66,12 @@
         (is (every? map? (vals (:members c1))))
         (is (apply (every-pred :name :modifiers)
                    (mapcat vals (vals (:members c1))))))
+      (testing "doesn't throw on classes without dots in classname"
```

Comment:
Not the most elegant solution, but that's the easiest way to instantiate a class with no dots in its name. @bbatsov, what do u think?

Reply:
Yeah, that's fine.